public/src/ux.js

```diff
@@ -144,6 +144,21 @@ export function setupUI() {
     // var dummyCounter=0;
 
 
+    // Function updates the values of project inputs accordingly
+    const watchOneInputChangeAndUpdateAnother = (objectToListen, objectToChange) => $(`${objectToListen}`).on('input', (e) => { $(`${objectToChange}`).val(e.target.value); });
```

Comment:
This is good, can you encapsulate all of this code into a function say `setupProjectNameListeners` and just call it here. This will be more readable and maintainable.

Reply:
Sure @satu0king